app/components/primer/beta/close_button.rb

```diff
@@ -28,7 +28,7 @@ def initialize(type: DEFAULT_TYPE, **system_arguments)
           "close-button",
           system_arguments[:classes]
         )
-        @system_arguments[:"aria-label"] ||= "Close"
+        @system_arguments[:"aria-label"] = aria("label", system_arguments) || "Close"
```

Comment:
The `aria` method gets the attribute from either a String or a nested hash:

https://github.com/primer/view_components/blob/79d2b02d264410e82df3711d05939cfb15b9efef/app/components/primer/component.rb#L49-L51